PyInstaller/utils/win32/versioninfo.py

```diff
@@ -101,7 +104,10 @@ def pefile_read_version(filename):
 
 
 def getRaw(o):
-    return str(buffer(o))
+    if sys.version_info >= (3, 0, 0):
+        return str.encode(o)
+    else:
+        return str(o)
 
```

Comment:
Create a new function in compat called `encode` in compat.py and then use that function to encode the string if on py3.